app/bundles/CoreBundle/Assets/css/app/less/components/forms.less

```diff
@@ -73,7 +73,7 @@ label {
 
   .search {
     border: 1px solid #fff;
-    background-color: transparent;
+    background-color: transparent !important;
```

Comment:
Nope nope nope. If you're having some issue with a style not coming through, you need to look at the cause and come up with a fix to override. It will either mean combining selectors or fixing something in the styling that is inherited.

Reply:
Oh boy, okay. If it's to handle inline code and it can't be fixed in another way than throwing more JS on it, it's okay to leave it in. I'd suggest adding an inline comment wherever there is an `!important` to explain what is being overridden and by what so future devs understand what's going on.

Reply:
Done!